taskoutput/task_output.go

```diff
@@ -25,7 +28,7 @@ func InitializeTaskOutput(env evergreen.Environment, opts TaskOptions) *TaskOutp
 	settings := env.Settings()
 
 	output := &TaskOutput{}
-	if settings.LoggerConfig.DefaultLogger != "buildlogger" {
+	if settings.LoggerConfig.DefaultLogger == "evergreen" || utility.StringSliceContains(settings.LoggerConfig.EvergreenLoggerProjects, opts.ProjectID) {
```

Comment:
Can you use model.EvergreenLogSender here directly or is there an import cycle? (I assume this is temporary in which case no need to re-organize if it does cycle)

Reply:
It causes an import cycle :(